ext/opcache/ZendAccelerator.c

```diff
@@ -3411,12 +3411,14 @@ void zend_accel_schedule_restart(zend_accel_restart_reason reason)
 	HANDLE_UNBLOCK_INTERRUPTIONS();
 }
 
-/* this is needed because on WIN32 lock is not decreased unless ZCG(counted) is set */
+static void accel_deactivate_now()
```

Comment:
inline?

Reply:
Adding `inline` will only have one effect: it will supress `-Wunused`, which I consider harmful.
This function is so small, the compiler will very likely inline it automatically without the `inline` hint, and if not, I'd trust this decision.
... but if you really want `inline`, I'll add it.

Reply:
It's ok, I'm not familiar with the rules compilers use to decide whether to inline so I trust you on that one :slightly_smiling_face:

Reply:
`inline` in plain C is (besides suppressing `-Wunused`) just a polite hint, which rarely has any effect on the generated code. The compiler can inline functions without `inline` at will, and can ignore the keyword as well. That's why there is `zend_always_inline` (GCC extension `__attribute((always_inline))`) which forces the GCC to really inline it, against GCC's own heuristics, for devs who think they know what they're doing.
(In C++, `inline` is slightly different; for example, it generates a weak symbol.)